package.json

```diff
@@ -51,8 +51,9 @@
     "lint:eslint:fix": "npm-run-all \"lint:eslint --fix\"",
     "lint:fix": "npm-run-all lint:*:fix",
     "test": "node bin/run-tests.js",
-    "test:blueprints:js": "EMBER_TYPESCRIPT_BLUEPRINTS=false pnpm test:blueprints:ts",
-    "test:blueprints:ts": "mocha node-tests/blueprints/**/*-test.js",
+    "test:blueprints:js": "EMBER_TYPESCRIPT_BLUEPRINTS=false pnpm test:blueprints:transpiled-js",
+    "test:blueprints:transpiled-js": "mocha node-tests/blueprints/**/*-test.js",
+    "test:blueprints:ts": "mocha node-tests/blueprints-ts/**/*-test.js",
```

Comment:
Splitting the .ts tests into a separate script seemed the easiest, but `:ts` was already used so I renamed the other one. Not sure about the naming though.